contrib/p4/p4execution.py

```diff
@@ -761,7 +751,8 @@ def create_switch_mount_copy(self, switch_name):
         os.mkdir(switch_path + "/P4")
         os.mkdir(switch_path + "/tables")
 
-        copy_tree(self.switch_source_path + "/log", switch_path + "/log")
+        # Create log file for switch to use
+        open(switch_path + "/log/switch_log.txt", "x")
```

Comment:
This leaks a file descriptor:
```suggestion
        open(switch_path + "/log/switch_log.txt", "x").close()
```